platforms/ide/problems-api/src/main/java/org/gradle/api/problems/internal/DefaultProblemReporter.java

```diff
@@ -57,30 +65,25 @@ public RuntimeException throwing(Action<ProblemSpec> spec) {
 
     public RuntimeException throwError(RuntimeException exception, Problem problem) {
```

Comment:
❓ Why is `throwError()` public and not `private`?

Reply:
Right, no reason for it to be public.